### PR TITLE
Refactored with pure css

### PR DESCRIPTION
**Pull request by emptycoffee**
Refactored into a zurb grid for easier maintainability.

### CODE REVIEW

**Comment by emptycoffee** [comment]
For obvious reasons, while previewing the file in github it doesn't render properly and the logos appear one per line, but jekyll will render it properly ;)

File: events/2018/sponsors.md
Comment (emptycoffee) [inline events/2018/sponsors.md:27]: This one requires the 'small-pull-6' class to move it to the left, otherwise it gets populated on the right. But also it can use 'end' See: https://foundation.zurb.com/sites/docs/grid.html#incomplete-rows

File: events/2018/sponsors.md
Comment (emptycoffee) [inline events/2018/sponsors.md:18]: Each row has to add 12 columns. This can be changed into 'small-6 large-4' to make it render differently in large/small screens. Having it done this way makes the job of adding or removing one logo much easier, but the class 'end' would have to be added to the last item of each grid to prevent it from rendering on the right leaving blank spaces.

**Comment by tophertimzen** [comment]
I'm okay with this change but I am also not the maintainer of ensuring logos are placed onto the site. @koronkowy and @btrimbo, you two have been doing this... does this PR make that easier?

**Review by tophertimzen — APPROVED** [review]
Approved and pending comment from the maintainers of `sponsors.md`

**Comment by koronkowy** [comment]
I was working on the initial changes with @emptycoffee to make what used to be on there work. This is the cleaner better coded version / much easier to maintain and will function better.

**Pull request merged**
